omisego-sdk/src/main/java/co/omisego/omisego/custom/retrofit2/adapter/OMGCaller.kt

```diff
@@ -9,21 +9,21 @@ package co.omisego.omisego.custom.retrofit2.adapter
 
 import co.omisego.omisego.constant.ErrorCode
 import co.omisego.omisego.constant.Versions
-import co.omisego.omisego.custom.Callback
+import co.omisego.omisego.custom.OMGCallback
 import co.omisego.omisego.exception.OMGAPIErrorException
 import co.omisego.omisego.model.APIError
 import co.omisego.omisego.model.OMGResponse
 import retrofit2.Call
 import retrofit2.Response
 import java.net.HttpURLConnection
 
-internal class OMGCaller<T>(private val call: Call<OMGResponse<T>>) : OMGCall<OMGResponse<T>> {
+internal class OMGCaller<T>(private val call: Call<T>) : OMGCall<T> {
     override fun cancel() = call.cancel()
     override fun clone() = OMGCaller(call.clone())
-    override fun execute(): Response<OMGResponse<T>> = call.execute()
-    override fun enqueue(callback: Callback<OMGResponse<T>>) {
-        call.enqueue(object : retrofit2.Callback<OMGResponse<T>> {
-            override fun onFailure(call: Call<OMGResponse<T>>, t: Throwable) {
+    override fun execute(): Response<T> = call.execute()
```

Comment:
This call should still return `OMGResponse<T>` I think, since Response is a retrofit class and not our custom class 🤔

Reply:
The `call` property should be a `Call<OMGResponse<T>>`.
Basically, you can see `OMGCall` as a `Call<OMGResponse<T>>` (but you can't actually declare it like this because of the callback haha)

Reply:
Fixed it!